grass.cpp

```diff
@@ -90,27 +104,25 @@ class GRASSDataset final : public GDALDataset
 {
     friend class GRASSRasterBand;
 
-    char *pszGisdbase;
-    char *pszLocation; /* LOCATION_NAME */
-    char *pszElement;  /* cellhd or group */
+    std::string pszGisdbase;
```

Comment:
The "psz" prefix stands for Pointer to a String Zero-terminated.  For a std::string, a better prefix is "os" for Object String